src/core/translatorstorageprivate.cpp

```diff
@@ -0,0 +1,7 @@
+#include "translatorstorageprivate.h"
+
+
+TranslatorStoragePrivate::TranslatorStoragePrivate(): translators(std::vector<std::unique_ptr<QTranslator>>())
```

Comment:
Move TranslatorStoragePrivate to "translatorstorage.cpp" use forward declaration.

Reply:
Moved it to the file translatestorage.h, I think it will be more clear